ros/src/waypoint_follower/src/pure_pursuit_core.cpp

```diff
@@ -234,9 +234,26 @@ bool PurePursuit::verifyFollowing() const
   double a = 0;
   double b = 0;
   double c = 0;
-  getLinearEquation(current_waypoints_.getWaypointPosition(1), current_waypoints_.getWaypointPosition(2), &a, &b, &c);
+  // edufford start
+  //getLinearEquation(current_waypoints_.getWaypointPosition(1), current_waypoints_.getWaypointPosition(2), &a, &b, &c);
+
+  // Use next 2 waypts from current pose to make a line seg to check
+  int next_wp = closest_waypoint_idx_ + 1;
```

Comment:
`closest_waypoint_idx_` is not initialized in the class declaration. Is it guaranteed that `getClosestWaypoint` will assign a value to `closest_waypoint_idx_` before it is used here?

Reply:
Good call, I will initialize closest_waypoint_idx_ in the declaration.

Reply:
Nice!